core/src/main/java/org/opensearch/sql/expression/operator/arthmetic/MathUtil.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.expression.operator.arthmetic;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
```

Comment:
This could be troublesome: https://github.com/opensearch-project/sql/pull/1045#discussion_r1022209834

Reply:
I'm not sure if MathUtil is necessary.  Let's include `private` function `truncateNumber` in `MathematicalFunction.java`.